extensions/certmanager/generator-v1/cmd/generate/generate.go

```diff
@@ -58,7 +58,7 @@ func main() {
 	// optional ApiGroup and ApiVersion for the go package (which is added to the generated java class)
 	packageMapping := map[string]schemagen.PackageInformation{
 		"github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1": {JavaPackage: "io.fabric8.certmanager.api.model.v1", ApiGroup: "cert-manager.io", ApiVersion: "v1"},
-		"github.com/cert-manager/cert-manager/pkg/apis/acme/v1":        {JavaPackage: "io.fabric8.certmanager.api.model.acme.v1", ApiGroup: "cert-manager.io", ApiVersion: "v1"},
+		"github.com/cert-manager/cert-manager/pkg/apis/acme/v1":        {JavaPackage: "io.fabric8.certmanager.api.model.acme.v1", ApiGroup: "acme.cert-manager.io", ApiVersion: "v1"},
 		"github.com/cert-manager/cert-manager/pkg/apis/meta/v1":        {JavaPackage: "io.fabric8.certmanager.api.model.meta.v1", ApiGroup: "cert-manager.io", ApiVersion: "v1"},
```

Comment:
Should this be changed as well? https://github.com/cert-manager/cert-manager/blob/ee8ec69fadff165afa96c2dd22264c16fdb7d065/pkg/apis/meta/doc.go#L22

The internal ones probably need updated as well, but they aren't contributing anything to our model correct?

Reply:
Umm, I didn't see any CRD in this apiGroup. All generated classes in this package don't seem to have any `@Group` annotation as well. We can add it for the sake of completeness

Reply:
Maybe we can do this in a separate issue.

Reply:
@shawkins about the meta-package I don't know if we need an update.

@rohanKanojia where is the `@Group` annotation missing? I can add that as well.